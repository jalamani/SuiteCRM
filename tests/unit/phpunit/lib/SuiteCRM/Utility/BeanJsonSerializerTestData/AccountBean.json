{
  "date_entered": "2018-06-12 11:01:34",
  "date_modified": "2018-06-12 11:01:34",
  "modified_user_id": 1,
  "assigned_user_id": "seed_max_id",
  "annual_revenue": "",
  "billing_address_street": "345 Sugar Blvd.",
  "billing_address_city": "Alabama",
  "billing_address_state": "CA",
  "billing_address_country": "USA",
  "billing_address_postalcode": 48357,
  "billing_address_street_2": "",
  "billing_address_street_3": "",
  "billing_address_street_4": "",
  "description": "",
  "email1": "hr.kid@example.tv",
  "email2": "kid.phone@example.co.jp",
  "email_opt_out": 0,
  "invalid_email": 0,
  "employees": "",
  "id": "11111111-1111-1111-1111-111111111111",
  "industry": "Construction",
  "name": "Super Star Holdings Inc",
  "ownership": "",
  "parent_id": "",
  "phone_alternate": "",
  "phone_fax": "",
  "phone_office": "(554) 356-2192",
  "rating": "",
  "shipping_address_street": "345 Sugar Blvd.",
  "shipping_address_city": "Alabama",
  "shipping_address_state": "CA",
  "shipping_address_country": "USA",
  "shipping_address_postalcode": 48357,
  "shipping_address_street_2": "",
  "shipping_address_street_3": "",
  "shipping_address_street_4": "",
  "campaign_id": "",
  "sic_code": "",
  "ticker_symbol": "",
  "account_type": "Customer",
  "website": "www.theinfo.cn",
  "created_by": 1,
  "created_by_name": "Administrator",
  "modified_by_name": "Administrator",
  "opportunity_id": null,
  "case_id": null,
  "contact_id": null,
  "task_id": null,
  "note_id": null,
  "meeting_id": null,
  "call_id": null,
  "email_id": null,
  "member_id": null,
  "parent_name": "",
  "assigned_user_name": "Max Jensen",
  "account_id": "",
  "account_name": "",
  "bug_id": "",
  "module_dir": "Accounts",
  "table_name": "accounts",
  "object_name": "Account",
  "importable": true,
  "new_schema": true,
  "additional_column_fields": [
    "assigned_user_name",
    "assigned_user_id",
    "opportunity_id",
    "bug_id",
    "case_id",
    "contact_id",
    "task_id",
    "note_id",
    "meeting_id",
    "call_id",
    "email_id",
    "parent_name",
    "member_id"
  ],
  "relationship_fields": {
    "opportunity_id": "opportunities",
    "bug_id": "bugs",
    "case_id": "cases",
    "contact_id": "contacts",
    "task_id": "tasks",
    "note_id": "notes",
    "meeting_id": "meetings",
    "call_id": "calls",
    "email_id": "emails",
    "member_id": "members",
    "project_id": "project"
  },
  "push_billing": null,
  "push_shipping": null,
  "new_with_id": false,
  "disable_vardefs": false,
  "new_assigned_user_name": null,
  "processed_dates_times": {
    "id": 1,
    "name": 1,
    "date_entered": 1,
    "date_modified": 1,
    "modified_user_id": 1,
    "modified_by_name": 1,
    "created_by": 1,
    "created_by_name": 1,
    "description": 1,
    "deleted": 1,
    "created_by_link": 1,
    "modified_user_link": 1,
    "assigned_user_id": 1,
    "assigned_user_name": 1,
    "assigned_user_link": 1,
    "SecurityGroups": 1,
    "account_type": 1,
    "industry": 1,
    "annual_revenue": 1,
    "phone_fax": 1,
    "billing_address_street": 1,
    "billing_address_street_2": 1,
    "billing_address_street_3": 1,
    "billing_address_street_4": 1,
    "billing_address_city": 1,
    "billing_address_state": 1,
    "billing_address_postalcode": 1,
    "billing_address_country": 1,
    "rating": 1,
    "phone_office": 1,
    "phone_alternate": 1,
    "website": 1,
    "ownership": 1,
    "employees": 1,
    "ticker_symbol": 1,
    "shipping_address_street": 1,
    "shipping_address_street_2": 1,
    "shipping_address_street_3": 1,
    "shipping_address_street_4": 1,
    "shipping_address_city": 1,
    "shipping_address_state": 1,
    "shipping_address_postalcode": 1,
    "shipping_address_country": 1,
    "email1": 1,
    "email_addresses_primary": 1,
    "email_addresses": 1,
    "email_addresses_non_primary": 1,
    "parent_id": 1,
    "sic_code": 1,
    "parent_name": 1,
    "members": 1,
    "member_of": 1,
    "email_opt_out": 1,
    "invalid_email": 1,
    "cases": 1,
    "email": 1,
    "tasks": 1,
    "notes": 1,
    "meetings": 1,
    "calls": 1,
    "emails": 1,
    "documents": 1,
    "bugs": 1,
    "contacts": 1,
    "opportunities": 1,
    "project": 1,
    "leads": 1,
    "campaigns": 1,
    "campaign_accounts": 1,
    "campaign_id": 1,
    "campaign_name": 1,
    "prospect_lists": 1,
    "aos_quotes": 1,
    "aos_invoices": 1,
    "aos_contracts": 1,
    "jjwg_maps_lng_c": 1,
    "jjwg_maps_lat_c": 1,
    "jjwg_maps_geocode_status_c": 1,
    "jjwg_maps_address_c": 1
  },
  "process_save_dates": true,
  "save_from_post": true,
  "duplicates_found": false,
  "deleted": 0,
  "update_date_modified": true,
  "update_modified_by": true,
  "update_date_entered": false,
  "set_created_by": true,
  "ungreedy_count": false,
  "module_name": "Accounts",
  "column_fields": [
    "id",
    "name",
    "date_entered",
    "date_modified",
    "modified_user_id",
    "modified_by_name",
    "created_by",
    "created_by_name",
    "description",
    "deleted",
    "created_by_link",
    "modified_user_link",
    "assigned_user_id",
    "assigned_user_name",
    "assigned_user_link",
    "SecurityGroups",
    "account_type",
    "industry",
    "annual_revenue",
    "phone_fax",
    "billing_address_street",
    "billing_address_street_2",
    "billing_address_street_3",
    "billing_address_street_4",
    "billing_address_city",
    "billing_address_state",
    "billing_address_postalcode",
    "billing_address_country",
    "rating",
    "phone_office",
    "phone_alternate",
    "website",
    "ownership",
    "employees",
    "ticker_symbol",
    "shipping_address_street",
    "shipping_address_street_2",
    "shipping_address_street_3",
    "shipping_address_street_4",
    "shipping_address_city",
    "shipping_address_state",
    "shipping_address_postalcode",
    "shipping_address_country",
    "email1",
    "email_addresses_primary",
    "email_addresses",
    "email_addresses_non_primary",
    "parent_id",
    "sic_code",
    "parent_name",
    "members",
    "member_of",
    "email_opt_out",
    "invalid_email",
    "cases",
    "email",
    "tasks",
    "notes",
    "meetings",
    "calls",
    "emails",
    "documents",
    "bugs",
    "contacts",
    "opportunities",
    "project",
    "leads",
    "campaigns",
    "campaign_accounts",
    "campaign_id",
    "campaign_name",
    "prospect_lists",
    "aos_quotes",
    "aos_invoices",
    "aos_contracts",
    "jjwg_maps_lng_c",
    "jjwg_maps_lat_c",
    "jjwg_maps_geocode_status_c",
    "jjwg_maps_address_c"
  ],
  "list_fields": [],
  "current_notify_user": null,
  "fetched_row": {
    "id": "11111111-1111-1111-1111-111111111111",
    "name": "Super Star Holdings Inc",
    "date_entered": "2018-06-12 11:01:34",
    "date_modified": "2018-06-12 11:01:34",
    "modified_user_id": 1,
    "created_by": 1,
    "description": null,
    "deleted": 0,
    "assigned_user_id": "seed_max_id",
    "account_type": "Customer",
    "industry": "Construction",
    "annual_revenue": null,
    "phone_fax": null,
    "billing_address_street": "345 Sugar Blvd.",
    "billing_address_city": "Alabama",
    "billing_address_state": "CA",
    "billing_address_postalcode": 48357,
    "billing_address_country": "USA",
    "rating": null,
    "phone_office": "(554) 356-2192",
    "phone_alternate": null,
    "website": "www.theinfo.cn",
    "ownership": null,
    "employees": null,
    "ticker_symbol": null,
    "shipping_address_street": "345 Sugar Blvd.",
    "shipping_address_city": "Alabama",
    "shipping_address_state": "CA",
    "shipping_address_postalcode": 48357,
    "shipping_address_country": "USA",
    "parent_id": null,
    "sic_code": null,
    "campaign_id": null,
    "id_c": null,
    "jjwg_maps_lng_c": null,
    "jjwg_maps_lat_c": null,
    "jjwg_maps_geocode_status_c": null,
    "jjwg_maps_address_c": null,
    "email1": "hr.kid@example.tv"
  },
  "fetched_rel_row": {
    "modified_by_name": "Administrator",
    "created_by_name": "Administrator",
    "assigned_user_id": "seed_max_id",
    "assigned_user_name": "Max Jensen",
    "parent_name": "",
    "campaign_name": ""
  },
  "layout_def": null,
  "force_load_details": false,
  "optimistic_lock": true,
  "disable_custom_fields": false,
  "number_formatting_done": false,
  "process_field_encrypted": false,
  "acltype": "module",
  "additional_meta_fields": [],
  "notify_inworkflow": null,
  "special_notification": false,
  "in_workflow": false,
  "tracker_visibility": true,
  "listview_inner_join": [],
  "in_import": false,
  "in_save": null,
  "logicHookDepth": {
    "before_retrieve": 0,
    "after_retrieve": 0
  },
  "max_logic_depth": 10,
  "loaded_relationships": [
    "member_of",
    "campaign_accounts"
  ],
  "is_updated_dependent_fields": false,
  "acl_category": null,
  "old_modified_by_name": null,
  "required_fields": {
    "id": 1,
    "name": 1
  },
  "acl_fields": true,
  "jjwg_maps_lng_c": "",
  "jjwg_maps_lat_c": "",
  "created_by_link": "",
  "modified_user_link": "",
  "assigned_user_link": "",
  "SecurityGroups": "",
  "email_addresses_primary": "",
  "email_addresses": "",
  "email_addresses_non_primary": "",
  "members": "",
  "cases": "",
  "email": "",
  "tasks": "",
  "notes": "",
  "meetings": "",
  "calls": "",
  "emails": "",
  "documents": "",
  "bugs": "",
  "contacts": "",
  "opportunities": "",
  "project": "",
  "leads": "",
  "campaigns": "",
  "campaign_name": "",
  "prospect_lists": "",
  "aos_quotes": "",
  "aos_invoices": "",
  "aos_contracts": "",
  "jjwg_maps_geocode_status_c": "",
  "jjwg_maps_address_c": "",
  "relDepth": 0,
  "rel_fields_before_value": {
    "opportunity_id": null,
    "bug_id": "",
    "case_id": null,
    "contact_id": null,
    "task_id": null,
    "note_id": null,
    "meeting_id": null,
    "call_id": null,
    "email_id": null,
    "member_id": null,
    "project_id": null
  }
}