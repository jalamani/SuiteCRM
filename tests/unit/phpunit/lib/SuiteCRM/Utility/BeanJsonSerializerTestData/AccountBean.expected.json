{
  "id": "11111111-1111-1111-1111-111111111111",
  "meta": {
    "created": {
      "date": "2018-06-12 11:01:34",
      "user_id": 1,
      "user_name": "Administrator"
    },
    "modified": {
      "date": "2018-06-12 11:01:34",
      "user_id": 1,
      "user_name": "Administrator"
    },
    "assigned": {
      "user_id": "seed_max_id",
      "user_name": "Max Jensen"
    }
  },
  "name": {
    "name": "Super Star Holdings Inc"
  },
  "description": "",
  "account_type": "Customer",
  "industry": "Construction",
  "annual_revenue": "",
  "phone": {
    "fax": "",
    "office": "(554) 356-2192",
    "alternate": ""
  },
  "address": {
    "billing": {
      "street": "345 Sugar Blvd.",
      "city": "Alabama",
      "state": "CA",
      "postalcode": 48357,
      "country": "USA"
    },
    "shipping": {
      "street": "345 Sugar Blvd.",
      "city": "Alabama",
      "state": "CA",
      "postalcode": 48357,
      "country": "USA"
    }
  },
  "rating": "",
  "website": "www.theinfo.cn",
  "ownership": "",
  "employees": "",
  "ticker_symbol": "",
  "parent": {
    "id": "",
    "name": ""
  },
  "sic_code": "",
  "campaign": {
    "id": "",
    "name": ""
  },
  "email": [
    "hr.kid@example.tv"
  ]
}